### PR TITLE
add usershell update

### PR DESCRIPTION
**Pull request by alichaddad**
https://github.com/threefoldtech/js-ng/issues/135

### CODE REVIEW

File: jumpscale/threesdk/threebot.py [hunk 1]
```diff
@@ -54,35 +60,40 @@ class ThreeBot(Container):
...
             words (str, optional): seed phrase of the user. Defaults to None.
             explorer (str, optional): which explorer network to use: mainnet, testnet, devnet. Defaults to None.
+            development (bool, optional): if true will mount codedir. Defaults to False.
 
         Raises:
             Value: Container with specified name already exists
             Value: explorer not in mainnet, testnet, devnet
         """
+        name = name or DEFAULT_CONTAINER_NAME
+        image = image or f"{DEFAULT_IMAGE}:{DEFAULT_TAG}"
```
Comment (grimpy) [inline jumpscale/threesdk/threebot.py:81]: we shouldnt use DEFAULT_TAG here but instead the version of our current 3sdk

File: jumpscale/entry_points/usershell.py [hunk 2]
```diff
@@ -32,10 +37,36 @@
...
+    for platform in resp["assets"]:
+        if sys.platform in platform["name"]:
+            download_link = platform["browser_download_url"]
+    return version, download_link
+
+
+def update():
+    print("checking for updates")
+    latest_version, binary_link = get_binary_link()
+    current_version = pkg_resources.get_distribution("js-ng").version
```
Comment (grimpy) [inline jumpscale/entry_points/usershell.py:55]: could we abstract this line somewhere in js-ng itself so we don't repeat this pattern? Just incase we change the way we version stuff in the future

File: docs/wiki/3sdk.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+# Working with 3sdk
+
+`3sdk` is a command line tool that provides the user with a simple interface to orchestrate his threebot containers.
+It allows the user to create a container that is ready to start a threebot server on with the user identity already registered on that container.
+
+## Using 3sdk
+
+You can get the binary from the [release page](https://github.com/threefoldtech/js-ng/releases). Make sure to change permission to allow the binary to be executed.
```
Comment (xmonader) [inline docs/wiki/3sdk.md:8]: can we add an example `chmod +x DOWNLOADED_BINARY_PATH`

File: docs/wiki/3sdk.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+The user is then asked to choose which explorer networks he is in:
+
+![SDK explorer](images/sdkexplorer.png)
+
+The container should be created now with default name `3bot-ng`.
+
+## Building the 3sdk
+
+You can get latest available binary by using `pyinstaller`. Which creates the `3sdk` binary based on `usershell.spec` file.
```
Comment (xmonader) [inline docs/wiki/3sdk.md:34]: can we change references of usershell.spec to maybe threesdk.spec instead?

**Pull request merged**